documentation/Troubleshooting.md

```diff
@@ -83,285 +92,391 @@ Traceback (most recent call last):
+### How to solve
+1. Add the connection user to the sudoers group on the affected cluster nodes. For example, on Ubuntu, use the following command (note that a reboot is required):
+   ```bash
+   sudo adduser <username> sudo
+   ```
+2. To allow the connection user to run `sudo` commands without requiring a password, edit the `/etc/sudoers` file and add the following line at the end:
+   ```bash
+   username  ALL=(ALL) NOPASSWD:ALL
+   ```
+   Replace `username` with the actual username of the connection user.
+3. Reboot the affected nodes and verify that the user has the required sudo privileges.
+4. Retry the `install` or `add_node` procedure.
 
-```bash
-sudo adduser <username> sudo
-```
+### Recommendations
+To prevent this issue in the future:
+- Ensure all connection users are properly configured with sudo privileges on all nodes before running any procedures.
+- Regularly audit the sudoer configurations to avoid permission issues during deployments or node additions.
 
-To run `sudo` commands without being asked for a password, add
-```bash
-username  ALL=(ALL) NOPASSWD:ALL
-```
-in the end of `/etc/sudoers` file, where `username` is a name of the connection user.
+>**Note**  
+>Not applicable.
 
 # Troubleshooting Tools
 
 This section describes the additional tools that Kubemarine provides for convenient troubleshooting of various issues.
 
 ## Etcdctl Script
 
-This script allows you to execute `etcdctl` queries without installing an additional binary file and setting up a connection. This file is installed during the `prepare.thirdparties` installation task on all control-planes and requires root privileges.
+### Description
+This script allows you to execute `etcdctl` queries without the need for additional binary installations or manual connection setup. The script is installed during the `prepare.thirdparties` task on all control-plane nodes and requires root privileges to execute commands.
 
-To execute a command through this script, make sure you meet all the following prerequisites:
+### Alerts
+Not applicable.
 
-* You run the command from the control-plane node with root privileges.
-* You have configured `admin.conf` on node.
-* The node with which you are running has all the necessary ETCD certificates and they are located in the correct paths.
+### Stack trace(s)
+Not applicable.
 
-If all prerequisites are achieved, you can execute almost any `etcdctl` command.
-For example:
+### How to solve
+To use the `etcdctl` script, ensure the following prerequisites are met:
+1. You are running the command from a control-plane node with root privileges.
+2. The `admin.conf` is configured on the node.
+3. The node has the necessary ETCD certificates located in the correct paths.
 
+Once the prerequisites are satisfied, you can execute most `etcdctl` commands. Examples include:
+```bash
+etcdctl member list
+etcdctl endpoint health --cluster -w table
+etcdctl endpoint status --cluster -w table
```

Comment:
This section is not about a particular trouble. I am not sure we need to format it as troubleshooting section. We probably can keep it as it was before